### PR TITLE
Change length of createdBy and lastModifiedBy property to 64 characters

### PR DESCRIPTION
**Pull request by melanieretter**
The length of the columns created_by and last_modified_by is increased to 64 characters.
 
Signed-off-by: Melanie Retter <melanie.retter@bosch-si.com>

### CODE REVIEW

**Comment by schabdo** [comment]
@hawkbit-bot verify please

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_7__add_rollout_approval_fields___DB2.sql [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ALTER TABLE sp_rollout ADD column approval_decided_by varchar(40);
-ALTER TABLE sp_rollout ADD column approval_remark varchar(255);
\ No newline at end of file
+ALTER TABLE sp_rollout ADD COLUMN approval_decided_by varchar(40);
```
Comment (laverman) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_7__add_rollout_approval_fields___DB2.sql]: Shouldn't it be 64 characters as well?

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/SQL_SERVER/V1_12_7__add_rollout_approval_fields___SQL_SERVER.sql [hunk 2]
```diff
@@ -1,2 +1,2 @@
-ALTER TABLE sp_rollout ADD column approval_decided_by varchar(40);
-ALTER TABLE sp_rollout ADD column approval_remark varchar(255);
\ No newline at end of file
+ALTER TABLE sp_rollout ADD approval_decided_by varchar(40);
```
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/SQL_SERVER/V1_12_7__add_rollout_approval_fields___SQL_SERVER.sql]: Shouldn't this be varchar(64)?

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_7__add_rollout_approval_fields___DB2.sql (hunk 1 above)
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_7__add_rollout_approval_fields___DB2.sql]: Shouldn't this be varchar(64)?

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_8__change_length_of_created_last_modified_by___DB2.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action ALTER COLUMN created_by SET DATA TYPE VARCHAR(64);
+ALTER TABLE sp_action ALTER COLUMN last_modified_by SET DATA TYPE VARCHAR(64);
```
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_8__change_length_of_created_last_modified_by___DB2.sql]: I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
   created_by SET DATA TYPE VARCHAR(64),
   last_modified_by SET DATA TYPE VARCHAR(64);

Comment (stefbehl) [reply hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_8__change_length_of_created_last_modified_by___DB2.sql]: Applies to all queries below...

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/H2/V1_12_8__change_length_of_created_last_modified_by___H2.sql [hunk 4]
```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action ALTER COLUMN created_by VARCHAR(64);
```
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/H2/V1_12_8__change_length_of_created_last_modified_by___H2.sql]: I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
created_by SET DATA TYPE VARCHAR(64),
last_modified_by SET DATA TYPE VARCHAR(64);

Not 100% sure if H2 supports that... could you pls check?
Thx

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/MYSQL/V1_12_8__change_length_of_created_last_modified_by___MYSQL.sql [hunk 5]
```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action MODIFY created_by VARCHAR(64);
```
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/MYSQL/V1_12_8__change_length_of_created_last_modified_by___MYSQL.sql]: I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
created_by SET DATA TYPE VARCHAR(64),
last_modified_by SET DATA TYPE VARCHAR(64);

File: hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/SQL_SERVER/V1_12_8__change_length_of_created_last_modified_by___SQL_SERVER.sql [hunk 6]
```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action ALTER COLUMN created_by VARCHAR(64);
```
Comment (stefbehl) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/SQL_SERVER/V1_12_8__change_length_of_created_last_modified_by___SQL_SERVER.sql]: I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
created_by SET DATA TYPE VARCHAR(64),
last_modified_by SET DATA TYPE VARCHAR(64);

File: hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/AbstractJpaBaseEntity.java [hunk 7]
```diff
@@ -67,7 +67,7 @@ public long getCreatedAt() {
 
     @Override
     @Access(AccessType.PROPERTY)
-    @Column(name = "created_by", insertable = true, updatable = false, nullable = false, length = 40)
+    @Column(name = "created_by", insertable = true, updatable = false, nullable = false, length = 64)
```
Comment (schabdo) [inline hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/AbstractJpaBaseEntity.java]: Please also adapt `approvalDecidedBy`JPA entity

Comment (melanieretter) [reply hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/AbstractJpaBaseEntity.java]: done

**Pull request merged**